server/routes/admin.js

```diff
@@ -75,8 +75,12 @@ router.route('/settingsData')
                     plugins.push(pConf);
             }
         });
+        let themes = [];
+        fs.readdirSync('./themes').forEach(file => {
```

Comment:
Don't hardcode path, use path.join with __dirname or something so it is relative to this file, not CWD

Reply:
Done that way as that's how the Plugins are done!
I blame @davidmckenzie

Reply:
Please just fix it, because this can result in some bugs if you are running server from different cwd than expected, it is best to make this robust as possible

Reply:
Hmmm replacing this line with `fs.readdirSync(path.join(__dirname, 'themes'))` breaks settings loading completely... 
Any ideas?

Reply:
Hmmm no i'm not... 

replacing this line with `fs.readdirSync(path.join(__dirname, 'themes')).forEach(file => {` doesn't work. 😢

Reply:
You sure, because here `path.join(__dirname, 'themes')` gives expected result. Where is `themes` directory ?

Reply:
Yup, i don't think it likes being called inside fs.readdirSync 

It works fine if i throw it out to the logger :|